locksmith/src/operations/eventOperations.ts

```diff
@@ -14,6 +16,22 @@ namespace EventOperations {
     return Event.create(event)
   }
 
+  export const update = async (event: EventCreation): Promise<any> => {
+    event.owner = Normalizer.ethereumAddress(event.owner)
+    event.lockAddress = Normalizer.ethereumAddress(event.lockAddress)
+    return await Event.update(event, {
+      where: {
+        lockAddress: {
+          [Op.eq]: ethJsUtil.toChecksumAddress(event.lockAddress),
```

Comment:
Lets us` Normalizer.ethereumAddress` it should functionally the be the same as `ethJsUtil.toChecksumAddress` but should help us to localize the conversion